src/common/classes/vector.h

```diff
@@ -118,6 +126,20 @@ class Vector
 		return data[count];
 	}
 
+	void push(const T* items, const FB_SIZE_T itemsSize)
+	{
+		fb_assert(count <= FB_MAX_SIZEOF - itemsSize);
+		fb_assert(count + itemsSize <= Capacity);
+		memcpy(data + count, items, sizeof(T) * itemsSize);
+		count += itemsSize;
+	}
+
+	void append(const T* items, const FB_SIZE_T itemsSize)
```

Comment:
In general I agree, but in this particular place - not. We have 2 ways/styles (don't know how to call it better) to work with Vector:
A. add/insert + remove
B. push + pop
Yes, push and add perform exactly same action but logically code looks better when in pair with push pop (not add) is used.

What about itemsSize - fixed.